topics/community/tutorials/tools_subdomains/tutorial.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: tutorial_hands_on
+
+title: Make your tools available on your subdomain
+questions:
+- How can a tool be added in a section ?
+- How can a section be added in a subdomain ?
+objectives:
+- Learn to manage the tool panel of a subdomain for both batch and interactive tools
+time_estimation: "30m"
+key_points:
+- Subdomain
+- Tool panel
+- Interactive tools
+tags:
+  - subdomain
+  - earth-system
+  - interactive-tools
+contributions:
+    authorship:
+        - Marie59
+    funding:
+        - gallantries
+        - fairease
+        - eurosciencegateway
+
+---
+This tutorial explains how to make your brand new tools, once they're published through a Pull Request ([check this tutorial]({%link topics/dev/tutorials/tool-from-scratch/tutorial.md %}) on how to build your tool from scracth), visible on your subdomain. Here we'll follow the example on how to make the tools visible on [Galaxy for earth System](https://earth-system.usegalaxy.eu/).
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Add your batch tool to the right section
+
+> <hands-on-title>Tool sections (for batch tools) </hands-on-title>
+>    - Clone the Github repository of the Galaxy server (of where your subdomain to be attached)
+>    - For Galaxy Europe, fork the [usegalaxy-eu-tools repo](https://github.com/usegalaxy-eu/usegalaxy-eu-tools/tree/master)
+>    - Create a branch on your fork
+>    - Browse through the different yaml files and select the one that correspond to your subdomain for earth system everything is made under the ecology owner. 
+>    - Once in the yaml you can add the section for your tool (you want to know what are the existing tool sections ? Go check the [ToolShed's categories](https://toolshed.g2.bx.psu.edu/)) 
```

Comment:
The sections are not from the TS. They are the sections defined in the servers. E.g. here: https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/templates/galaxy/config/tool_conf.xml.j2#L640